pages/agent/v3/aws.md.erb

```diff
@@ -7,11 +7,15 @@ The Buildkite Agent can be run on AWS using our Elastic CI Stack for AWS.
 ## Install the Agent on your own AWS instance
 
 To run the agent on your own AWS instances use whichever installer matches your
-instance type. For Amazon Linux use the [Red Hat/CentOS installer](/docs/agent/v3/redhat).
+instance type. For Amazon Linux 2 use the
+[Red Hat/CentOS installer](/docs/agent/v3/redhat).
 
-## Use our pre-built Elastic CI Stack for AWS
```

Comment:
I didn’t think we needed to say pre-built here and below

Reply:
:100: